src/chksum.c

```diff
@@ -42,22 +54,46 @@ solv_chksum_create(Id type)
   switch(type)
     {
     case REPOKEY_TYPE_MD5:
+#ifdef WITH_OPENSSL
```

Comment:
why not to do on top of the file something like `#define solv_MD5_Init MD5_init`?

Reply:
@ignatenkobrain OK I changed it.